pkgs/development/embedded/avrdude/default.nix

```diff
@@ -19,9 +23,10 @@ stdenv.mkDerivation rec {
     texliveMedium
     texinfo
     texi2html
-  ];
+  ] ++ lib.optionals (!lib.meta.availableOn stdenv.hostPlatform elfutils) [ libelf ];
```

Comment:
Not sure why we need it in nativeBuildInputs here.

Reply:
I think I messed this up. The vendored copy of libelf only builds as a static library, and I thought that static inputs go in `nativeBuildInputs`. Upon thinking about it, that's not true, and would break cross-compilation. I'll land it in `buildInputs`.